index.html

```diff
@@ -339,5 +339,70 @@ <h3 class="card__topic">Skills</h3>
                 </div>
             </section>
         </main>
+        <footer class="footer">
+            <h2 class="footer__title">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h2>
+            <form class="footer__form" action="https://formspree.io/f/xqkjrbba" method="POST">
+                <input type="text" placeholder="Full name" name="name" id="name" maxlength="30" required />
+                <input type="email" placeholder="Email address" name="email" id="email" required />
+                <textarea
+                    placeholder="Enter comments here"
+                    name="description"
+                    id="description"
+                    maxlength="500"
+                    cols="30"
+                    rows="10"
+                    required
+                ></textarea>
```

Comment:
- [ ] Great work implementing most of the requirements for this milestone. It is impressive. To make it better, kindly reduce the value given to the `row` attribute for the `textarea` element. This would make your design identical to that of the Figma template you chose. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/206035929-e4ae2d4e-79ae-4e01-b434-d8804ed4bbc0.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/206035787-59dfd8e0-df25-47e1-a9b9-90e15d9166ab.png)